Gemfile

```diff
@@ -81,8 +82,7 @@ gem "sys-filesystem",                 "~>1.3.4"
 gem "terminal",                                        :require => false
 
 # Modified gems (forked on Github)
-gem "rugged",                         "=0.28.2.2", :source => "https://rubygems.manageiq.org", :require => false
-gem "ruport",                         "=1.7.0.3",  :source => "https://rubygems.manageiq.org"
+gem "ruport", "=1.7.0.3", :source => "https://rubygems.manageiq.org"
```

Comment:
Can you keep this alignment to how it was?  I know `rubocop` is going to complain, but it is matching the spacing from the `gem` calls above it.

Reply:
Ok done.